### ISSUE TITLE
[a11y] "Please select if any apply to your report" should have an h4 tag

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#526]
On the review page, "Please select if any apply to your report" has an H4 tag but it's not describing a heading or category/question.

I would either remove the text all together or remove the `H4` tag and just css to style it that way.

**Comment by Jkrzy** [comment]
Removing makes a-lot of sense here as there is no usable "select"functionality.

**Comment by Jacklynn** [comment]
Thanks!

and ack, noticed the confusing typo in the title "Should**n't** have"
